### ISSUE TITLE
Pashword causes Bromite to get stuck. Bromite unable to handle Scrypt algorithm 

### ISSUE THREAD

**Issue by NayamAmarshe** [bromite/bromite#2240]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser; not the website nor F-Droid nor anything else.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

103.0.5060.140

### Device architecture

arm64

### Android version

12.1

### Device model

Google Pixel 6A

### Changed flags

none

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No

### Describe the bug

Generating a hash on https://pashword.app causes bromite to freeze for 10 seconds, and on other devices even as long as a minute.
This bug doesn't happen on Brave.

### Steps to reproduce the bug

1. Go to https://pashword.app
2. Enter random details
3. Click get pashword.

### Expected behavior

It should take only 0.5 to 1 second to generate the pashword using scrypt.

**Comment by n-buna404** [comment]
Hi NayamAmarshe. You can go to site setting and enable JavaScript JIT for the site which was blocked by default globally.
This should improve the performance to generate the hashed password. Around 3 seconds on MiA1 but the performance may differ on different hardware.

**Comment by NayamAmarshe** [comment]
Thanks for the solution! I'll put this up in the FAQ😄
